src/register/Register.jsx

```diff
@@ -80,15 +81,16 @@ export default function Register() {
                                     required
                                 />
                             </p>
-                            {!isValidName && (
-                                <span className="error">Must be between 2-30 characters long. Can only contain alphabets, whitespace and dash '-'</span>
-                            )}
+                         <div>{!isValidName && (
+                                <span className="error" title="errorname" aria-label="errorname" role="alert">Must be between 2-30 characters long. Can only contain alphabets, whitespace and dash '-'</span>
+                             )}</div>
                         </form-group>
                         <div><br></br></div>
                         <form-group controlId="formUserame">
                             <p className="input-control">
-                                <label htmlFor="username">Username :</label>
-                                <input className="field"
+                                <label Id="Username">Username :</label>
```

Comment:
id not Id (lower case i). This is pointed out by the error message. Check other place in the code as well.